modules/standard/Buffers.chpl

```diff
@@ -772,5 +775,5 @@ module Buffers {
     if err then try ioerror(err, "in buffer.copyin");
     return ret;
   }
-}
+  }
```

Comment:
Why add a space at the end of the file?

Reply:
Oops, sorry that slipped in... I had a mis-matched curly bracket at some point and when trying to track it down, it seems I forgot to put things back as they were...  My confidence was sufficiently low that this would pass review that I didn't scrutinize the diff as much as I should've.